packages/myst-frontmatter/tests/authors.yml

```diff
@@ -50,50 +70,167 @@ cases:
+        - name: Just A. Name
+          nameParsed:
+            display: Just A. Name
+            given: Just A.
+            family: Name
           email: example@example.com
           corresponding: true
   - title: First email is corresponding (unless false)
     raw:
       author:
-        name: Someone
+        name: Just A. Name
         email: example@example.com
         corresponding: false
     normalized:
       authors:
-        - name: Someone
+        - name: Just A. Name
+          nameParsed:
+            display: Just A. Name
+            given: Just A.
+            family: Name
           email: example@example.com
           corresponding: false
   - title: Respect corresponding flag
     raw:
       authors:
-        - name: Someone
+        - name: Just A. Name
           email: example@example.com
-        - name: Next
+        - name: A. Nother Name
           email: example@example.com
           corresponding: true
     normalized:
       authors:
-        - name: Someone
+        - name: Just A. Name
+          nameParsed:
+            display: Just A. Name
+            given: Just A.
+            family: Name
           email: example@example.com
-        - name: Next
+        - name: A. Nother Name
+          nameParsed:
+            display: A. Nother Name
+            given: A. Nother
+            family: Name
           email: example@example.com
           corresponding: true
   - title: Warn if author has no name
     raw:
       authors:
-        - name: Someone
+        - name: Just A. Name
           email: example@example.com
         - email: example@example.com
           corresponding: true
     normalized:
       authors:
-        - name: Someone
+        - name: Just A. Name
+          nameParsed:
+            display: Just A. Name
+            given: Just A.
+            family: Name
           email: example@example.com
         - email: example@example.com
           corresponding: true
     warnings: 1
+  - title: Warn if author has no family name
+    raw:
+      authors:
+        - name:
+            given: Someone
+    normalized:
+      authors:
+        - name: ', Someone'
+          nameParsed:
+            display: ', Someone'
+            given: Someone
+    warnings: 1
+  - title: Warn if author has no given name
+    raw:
+      authors:
+        - name: Someone
+    normalized:
+      authors:
+        - name: 'Someone'
+          nameParsed:
+            display: 'Someone'
+            family: Someone
+    warnings: 1
+  - title: Parsed name is maintained
+    raw:
+      authors:
+        - name:
+            display: Just A. Name
+            given: Little
+            particle: von
+            family: Junior
+            suffix: Jr.
+    normalized:
+      authors:
+        - name: Just A. Name
+          nameParsed:
+            display: Just A. Name
+            given: Little
+            particle: von
+            family: Junior
+            suffix: Jr.
```

Comment:
not clear on the meaning of this test? it leaves the normalized in some sort of mismatched state?